tree/master/cloud/src/deployments/Static Content/Matter Center Assets/Document Landing Assets/Scripts/documentDetails.js

```diff
@@ -599,6 +600,83 @@ var oDocumentLandingObject = (function () {
             }
         }
     };
+    
+    oCommonFunctions.retrieveAdditionalMatterPropertyContentType = function (oData) {
+        // Wait for getCurrentUserTitle to complete
+        if (!oGlobalConstants.sUserLoginName) {
+            setTimeout(function () { oCommonFunctions.retrieveListItems(); }, 1000);
+        } else {
+            if (oGlobalConstants.sUserLoginName) {
+   				getallContentTypes(oData);
+   	        } else {
+                onQueryFailed(oGlobalConstants.sOperationPin);
+            }
+        }
+    };
+    
+    function getallContentTypes(oData) {
```

Comment:
Purpose: To retrieve all Content types so we can filter 'Additional Matter Properties' contenttype and get site columns Internal Name and Display Name for it